drivers/hmis/app/views/hmis_admin/access_overviews/index.haml

```diff
@@ -0,0 +1,92 @@
+- content_for :title, 'Access Overview'
+.d-flex
+  %h1= content_for :title
+= render 'menus/admin_tabs'
+.access-overview
```

Comment:
It feels funny that this page has a tiny gutter on the left, but bigger spacing between the cards and on the right. But maybe that is consistent with warehouse page design.

Reply:
I might be missing something, but it looks like it matches the warehouse design (such that it is) to me.